src/coreclr/jit/hwintrinsiclistarm64.h

```diff
@@ -469,6 +469,7 @@ HARDWARE_INTRINSIC(AdvSimd,       StoreSelectedScalar,
 HARDWARE_INTRINSIC(AdvSimd,       StoreSelectedScalarVector64x2,                                     8,      3,      true,  {INS_st2,            INS_st2,            INS_st2,            INS_st2,            INS_st2,            INS_st2,            INS_invalid,        INS_invalid,        INS_st2,            INS_invalid},     HW_Category_MemoryStore,           HW_Flag_BaseTypeFromFirstArg|HW_Flag_HasImmediateOperand|HW_Flag_SIMDScalar|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_NeedsConsecutiveRegisters)
 HARDWARE_INTRINSIC(AdvSimd,       StoreSelectedScalarVector64x3,                                     8,      3,      true,  {INS_st3,            INS_st3,            INS_st3,            INS_st3,            INS_st3,            INS_st3,            INS_invalid,        INS_invalid,        INS_st3,            INS_invalid},     HW_Category_MemoryStore,           HW_Flag_BaseTypeFromFirstArg|HW_Flag_HasImmediateOperand|HW_Flag_SIMDScalar|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_NeedsConsecutiveRegisters)
 HARDWARE_INTRINSIC(AdvSimd,       StoreSelectedScalarVector64x4,                                     8,      3,      true,  {INS_st4,            INS_st4,            INS_st4,            INS_st4,            INS_st4,            INS_st4,            INS_invalid,        INS_invalid,        INS_st4,            INS_invalid},     HW_Category_MemoryStore,           HW_Flag_BaseTypeFromFirstArg|HW_Flag_HasImmediateOperand|HW_Flag_SIMDScalar|HW_Flag_SpecialCodeGen|HW_Flag_SpecialImport|HW_Flag_NeedsConsecutiveRegisters)
+HARDWARE_INTRINSIC(AdvSimd,       StoreVectorAndZip,                                                 8,      2,     false,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_MemoryStore,           HW_Flag_BaseTypeFromFirstArg|HW_Flag_SpecialImport|HW_Flag_SpecialCodeGen)
```

Comment:
As mentioned above, with `SpecialCodeGen` flag for `StoreVectorAndZip`, we should be able to eliminate `StoreSelectedScalarVector64x*` and `StoreSelectedScalarVector128x*`.

Reply:
Something similar to `NI_AdvSimd_VectorTableLookup`

Reply:
Done